pkg/storage/unified/resource/document.go

```diff
@@ -107,10 +107,22 @@ type IndexableDocument struct {
 	// When the resource is managed by an upstream repository
 	Manager *utils.ManagerProperties `json:"manager,omitempty"`
 
+	// indexed only field for faceting manager info
+	ManagedBy string `json:"managedBy,omitempty"`
+
 	// When the manager knows about file paths
 	Source *utils.SourceProperties `json:"source,omitempty"`
 }
 
+func (m *IndexableDocument) UpdateCopyFields() *IndexableDocument {
```

Comment:
@owensmallwood @scottlepp -- rather than have each field need to manage its own copies this sets up a function that will be called before indexing that copies for us.